### ISSUE TITLE
Add a blocklist of URL's

### ISSUE THREAD

**Issue by PaulKinlan** [ebidel/devwebfeed#14]
There's probably some url sets that we don't want to allow to get posted such as googleplex etc. Might be good to add a configurable filter that stops us from doing something stupid.

**Comment by PaulKinlan** [comment]
Actually, rather than a URL list that could also leak things, maybe we should try and see if an external service can get a 200 response and similar data to what the extension sees.

**Comment by ebidel** [comment]
Good idea. Better to play it safe. If the crx makes a fetch without cookies
and gets a 200 back, that may work. Although if someone is on a corp IP it
may still go through....
